soap/serializers.py

```
@@ -0,0 +1,33 @@
+
+class SOAPReportSerializer(serializers.ModelSerializer):
+    """
+    خروجی استاندارد گزارش SOAP
+    """
+
+    class Meta:
+        model = SOAPReport
+        fields = [
+            'id', 'encounter_id', 'patient', 'doctor', 'subjective', 'objective',
+            'assessment', 'plan', 'metadata', 'generated_at', 'updated_at'
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)